app/src/main/java/com/marcossalto/targetmvd/network/models/UserSerializer.kt

```diff
@@ -8,29 +8,27 @@ data class User (
     @Json(name = "uid") val uid: String,
     @Json(name = "provider") val provider: String,
     @Json(name = "avatar") val avatar: Avatar,
-    @Json(name = "first_name") var firstName: String,
-    @Json(name = "last_name") var lastName: String,
+    @Json(name = "first_name") var first_name: String?,
+    @Json(name = "last_name") var last_name: String?,
```

Comment:
why changing the var names?

Reply:
Names of top-level or object properties which hold objects with behavior or mutable data should use camel-case names.
please check:
https://github.com/rootstrap/tech-guides/tree/master/kotlin#property-names

Reply:
> why changing the var names?

This change was related to KotlinJsonAdapter in Moshi. I fixed it